lib/excon/constants.rb

```diff
@@ -134,41 +135,42 @@ module WaitWritable; end
-    :ciphers              => 'ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:ECDHE-ECDSA-DES-CBC3-SHA:ECDHE-RSA-DES-CBC3-SHA:EDH-RSA-DES-CBC3-SHA:DES-CBC3-SHA:!DSS',
-    :connect_timeout      => 60,
-    :debug_request        => false,
-    :debug_response       => false,
-    :headers              => {
+    :ciphers             => 'ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:ECDHE-ECDSA-DES-CBC3-SHA:ECDHE-RSA-DES-CBC3-SHA:EDH-RSA-DES-CBC3-SHA:DES-CBC3-SHA:!DSS',
+    :connect_timeout     => 60,
+    :debug_request       => false,
+    :debug_response      => false,
+    :headers             => {
       'User-Agent' => USER_AGENT,
       'Accept'     =>  '*/*'
     },
-    :idempotent           => false,
-    :instrumentor_name    => 'excon',
-    :middlewares          => [
+    :idempotent          => false,
+    :instrumentor_name   => 'excon',
+    :middlewares         => [
       Excon::Middleware::ResponseParser,
       Excon::Middleware::Expects,
       Excon::Middleware::Idempotent,
       Excon::Middleware::Instrumentor,
       Excon::Middleware::Mock
     ],
-    :mock                 => false,
-    :nonblock             => true,
-    :omit_default_port    => false,
-    :persistent           => false,
-    :read_timeout         => 60,
-    :retry_errors         => DEFAULT_RETRY_ERRORS,
-    :retry_limit          => DEFAULT_RETRY_LIMIT,
-    :ssl_verify_peer      => true,
-    :ssl_uri_schemes      => [HTTPS],
-    :stubs                => :global,
-    :tcp_nodelay          => false,
-    :thread_safe_sockets  => true,
-    :uri_parser           => URI,
-    :versions             => VERSIONS,
-    :write_timeout        => 60
+    :mock                => false,
+    :nonblock            => true,
+    :omit_default_port   => false,
+    :persistent          => false,
+    :read_timeout        => 60,
+    :dns_timeouts        => 60,
```

Comment:
Really minor nitpick, but could you place this key alphabetically in the list? (that should be how all the existing keys are and can make it easier to find stuff in the future).